docs/introduction.rst

```diff
@@ -6,7 +6,7 @@ Readers to read content into a CaptionSet object,
 and then use one of the Writers to output the CaptionSet into
 captions of your desired format.
 
-Requires Python 2.7.
+Requires Python 3.6.
```

Comment:
I think we should mention it goes up to 3.9

Reply:
I presumed this will be treated as a minimum requirement, but your thinking is right, this isn't explicit, would `Python 3.6+, up to Python 3.9` work instead?
And also, should we limit the python version required in setup as well, to be `<3.10`?

Reply:
Yes, sounds good.

Reply:
Done in 3313204e15bd252229cf452badc0cec6e91424b2, with a slight change, of using braces instead of comma for specifying the required version in the docs.